web-components/src/components/cards/ReviewCard/ReviewCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box } from '@mui/material';
+
+import { Label } from '../../typography';
+import OnBoardingCard from '../OnBoardingCard';
+import { EditButton } from '../../buttons/EditButton';
+
+export interface ReviewCardProps {
+  title: string;
+  onEditClick: () => void;
+}
+
+const ReviewCard: React.FC<ReviewCardProps> = ({
+  title,
+  onEditClick,
+  children,
+}) => {
+  return (
+    <OnBoardingCard
+      sx={[
+        {
+          mt: 5,
+          mb: 0,
+        },
+      ]}
+    >
```

Comment:
```suggestion
export interface ReviewCardProps {
  title: string;
  onEditClick: () => void;
  sx?: SxProps<Theme>;
}
const ReviewCard: React.FC<ReviewCardProps> = ({
  title,
  onEditClick,
  children,
  sx,
}) => {
  return (
    <OnBoardingCard sx={sx}>
```
(then pass `sx={{ mt: 5, mb: 0 }}` where you were using -- though I think you might have to wrap these in array to take effect because of the issue I mentioned above)